opentaxii/persistence/sqldb/models.py

```diff
@@ -51,7 +52,9 @@ class ContentBlock(AbstractModel):
             'inbox_messages.id', onupdate='CASCADE', ondelete='CASCADE'),
         nullable=True)
 
-    content = schema.Column(types.LargeBinary, nullable=False)
+    content_type = types.LargeBinary()
+    content_type = content_type.with_variant(mysql.MEDIUMBLOB(), 'mysql')
```

Comment:
class-level is not the best place for these values. could they be transformed into module-level constants at the beginning of the file?

Reply:
hello @traut.

Please specify: do you want me to define a constant at the beginning of the models.py file, like:
```python
MYSQL_LARGE_BINARY = mysql.MEDIUMBLOB()
```

and later do something like this:
```python
content_type = types.LargeBinary()
content_type = content_type.with_variant(MYSQL_LARGE_BINARY, 'mysql')
```

?

Reply:
@zentavr Yeah, that's exactly what he means.

Reply:
Not fully. another comment I had -- I do not like the reliance on sequential execution (`content_type` field is defined and overloaded in the next line)

can't it be just
```python
content_type = types.LargeBinary().with_variant(MYSQL_LARGE_BINARY, 'mysql')
```
instead of 2 lines?

Reply:
@traut - fixed.